index.html

```diff
@@ -1592,17 +1590,40 @@ <h3 id="applying">
         <div class="issue" data-number="446"></div>
       </section>
     </section>
-    <section>
+    <section data-dfn-for="webappmanifest" data-link-for="webappmanifest">
       <h2>
         Manifest and its members
       </h2>
       <p>
         A <dfn>manifest</dfn> is a JSON document that contains startup
         parameters and application defaults for when a web application is
-        launched. A manifest consists of a top-level <a>object</a> that
-        contains zero or more members. Each of the members are defined below,
-        as well as how their values are processed.
+        launched. A manifest consists of a top-level <dfn>WebAppManifest</dfn>
+        <a>object</a> that contains zero or more members.
+        Each of the members are defined below, as well as how their values are processed.
```

Comment:
Nit: make sure you run tidy over the doc.

Reply:
Any info on this tidy and how to install/run it?